src/actinia_core/processing/actinia_processing/ephemeral_processing.py

```diff
@@ -1203,6 +1204,154 @@ def _cleanup(self):
             and os.path.isdir(self.temp_grass_data_base)
         ):
             shutil.rmtree(self.temp_grass_data_base, ignore_errors=True)
+        if self.ginit_tmpfiles:
+            for tmpfile in self.ginit_tmpfiles:
+                try:
+                    os.remove(tmpfile)
+                except Exception as e:
+                    self.message_logger.debug(
+                        f"Temporary file {tmpfile} can't be removed: {e}"
+                    )
+
+    def _check_pixellimit_rimport(self, process_executable_params):
+        """Check the current r.import command against the user cell limit.
+
+        Raises:
+            This method will raise an AsyncProcessError exception
+
+        """
+        rimport_inp = [x for x in process_executable_params if "input=" in x][
```

Comment:
At this point, an importer command would already be translated to `r.import`, right? So the following would be parsed correctly because it is already transformed?:
```
 {
            "id": "importer_1",
            "module": "importer",
            "inputs": [
                {
                    "import_descr": {
                        "source": "https://raw.githubusercontent.com/mmacata/pagestest/gh-pages/bonn.geojson",
                        "type": "vector"
                    },
                    "param": "map",
                    "value": "polygon"
                }
            ]
        },
```

(Maybe add an importer PC to the tests below?) :upside_down_face:

Reply:
Yes it is already transformed at this point. I also added a test for the importer